src/Symfony/Component/Security/Core/Validator/Constraints/UserPasswordValidator.php

```diff
@@ -39,6 +39,10 @@ public function validate($password, Constraint $constraint)
             throw new UnexpectedTypeException($constraint, __NAMESPACE__.'\UserPassword');
         }
 
+        if (null === $password || '' === $password) {
+            return;
```

Comment:
From documentation: `Emits an E_WARNING message when either of the supplied parameters is not a string.`.